utilities/ovs-p4ctl.in

```diff
@@ -953,15 +953,17 @@ def validate_args(argv, command, expected_nr):
         raise Exception("ovs-p4ctl: '{}' command requires at least {} "
                         "arguments".format(command, expected_nr))
 
-
+grpc_cl = ""
 def main():
+    global grpc_cl
     if len(sys.argv) < 2:
         print("ovs-p4ctl: missing command name; use --help for help")
         sys.exit(1)
+
     parser = argparse.ArgumentParser(usage=USAGE)
     parser.add_argument('command', help='Subcommand to run')
-
-    args = parser.parse_args(sys.argv[1:2])
+    grpc_cl = sys.argv[1]
```

Comment:
Need additional enhancements
1. Include the logic for IP validation once after parsing the parameter
2. If this parameter not provided, make sure it will be initialized with default address and same used for connectivity.